### DISCUSSION TITLE
Difference between DeepARE and DeepVAR Estimator

### DISCUSSION THREAD

**Discussion by miteshyadav** [awslabs/gluonts#1472] [Q&A] [answered]
According to docs, DeepVAR is a multi-variate version of DeepARE. However, DEEPARE accepts multiple inputs using the feat_dynamic_real and feat_static_cat parameters. What are the differences between both of these models and which should be used when?

**Comment by lostella** [comment, marked as answer]
@miteshyadav  `feat_dynamic_real` and `feat_static_cat` are input features that can be provided together with the `target` sequence. By "multivariate", we mean that the *target sequence* has multiple dimensions (it has an additional axis other than time, usually of length > 1).

In short, you should think about the `"target"` field in your data:

```python
# univariate
"target": [1.0, 2.0, 3.0, 4.0...]

# multivariate (of dimension 2 in this example)
"target": [[1.0, 2.0, 3.0, 4.0...], [5.0, 6.0, 7.0, 8.0, ...]]
```

In this sense, both DeepAR and DeepVAR can handle multivariate target sequences, since DeepAR can be configured to handle multivariate output e.g. using the `MultivariateGaussianOutput` class. However, DeepVAR was conceived specifically to be better able at dealing with the covariance of the target values, see [the associated paper](https://papers.nips.cc/paper/2019/hash/0b105cf1504c4e241fcc6d519ea962fb-Abstract.html), so it is expected to work better in the multivariate setting.
